### DISCUSSION TITLE
Provide GMP bindings as V module

### DISCUSSION THREAD

**Discussion by teaalltr** [vlang/v#11930] [Ideas]
https://gmplib.org/

Instead or as a complement to #11465

**Comment by dumblob** [comment]
Please not as part of the standard library - for that purpose it's insufficient (I'm out of time, but on request can provide links proving GNU MP inferiority compared to contemporary high-precision methods of calculation using binary floating points).

GNU MP would be nice as a binding (i.e. *V* module).

**Comment by teaalltr** [comment]
@dumblob thanks! P.S. i'm curious to take a look at those links 😄

**Comment by dumblob** [comment]
See https://github.com/vlang/vsl/issues/65 .

And if you'll be fine with that feel free to rename this issue to "provide GMP bindings as *V* module" or just close it :wink:.

**Comment by teaalltr** [comment]
@dumblob sure!
